README.md

```diff
@@ -54,8 +57,9 @@ Shaderc into.
 ```sh
 git clone https://github.com/google/shaderc $SOURCE_DIR
 cd $SOURCE_DIR/third_party
-git clone https://github.com/google/googletest.git
-git clone https://github.com/google/glslang glslang
+git clone https://github.com/google/googletest.git googletest
+git clone https://github.com/google/glslang.git glslang
+git clone https://github.com/KhronosGroup/SPIRV-Tools.git third_party/spirv-tools
```

Comment:
Also, why do we need the last argument on the previous three lines?  Were the instructions wrong before?